### ISSUE TITLE
Debug error

### ISSUE THREAD

**Issue by mralperem** [kakopappa/flutter_smartconfig_with_aes_cbc_encryption#1]
Hello,

When i try to compile the code to crate the apk i get an error saying "SimplePermissionsPlugin.java:9: error: cannot find symbol". How I can fix it?

**Comment by kakopappa** [comment]
Did you run pub get? Sounds like dependencies are missing

**Comment by mralperem** [comment]
Yes i try to run pub get but i have the error like so when i try to compile.

![Untitled](https://user-images.githubusercontent.com/29342713/134802425-14f47493-1885-4dbc-acc5-0b3c2f188b18.png)

**Comment by mralperem** [comment]
I found the problem it looks like the simple_permisson is no longer supported and deprecated. Could you please tell me how you were able to compile the program? Is there any work through around this problem?

**Comment by kakopappa** [comment]
Maybe because I am on flutter 2.2.3.

simple_permission is used to request/grant location permission. you can
replace it with any package that allows you to grant permission.

**Comment by mralperem** [comment]
Thank you for the information. I am going to migrate the package and chance the grant permisson with some other package and then send you for the review. Could you please share the apk file that you created so i can test it out?

**Comment by kakopappa** [comment]
created a new branch with updated code
https://github.com/kakopappa/flutter_smartconfig_with_aes_cbc_encryption/tree/v2

**Comment by mralperem** [comment]
Thank you for the update and great demonstration. Everything is working now
